input/blog/posts/net-foundation-election-2021-nominations-open.md

```diff
@@ -0,0 +1,16 @@
+---
+Title: .NET Foundation Election 2021 Nominations are Open!
+Author: Iris Classon
+Published: 2021-07-14
+---
+
+## .NET Foundation Election 2021 Nominations are Open!
+
+It’s that time of the year again dear friends- the .NET nominations! On behalf of the board and nomination committee we would like to welcome all of you to participate- be that by [applying as a candidate yourself](https://forms.office.com/Pages/ResponsePage.aspx?id=8ru98HbNZ0mMTFCoAnRpEmp3qxxG5iNBl2dHcSfTzztUOEdKT05ENERURFFIMTFOSDU3TE81NFJDQi4u), suggesting somebody else or simply preparing to vote and help spread the word.
+
+As a board member you get to influence and shape the future of one of the biggest platforms there is, .NET and everything it entails. You won’t be doing this alone, you’ll be working side by side with other passionate souls on the board, committees, TSG, Microsoft, community members and more. The [expectations for board members are outlined here](https://dotnetfoundation.sharepoint.com/:b:/g/EVpjN2ykcKJGkfPjbwbTNIUB_WOVtDAEElC-xXCmEh5QzQ)
+
+We’ve made a few changes over the years as we’ve learned and grown with the help of the community, and last year we started with a staggering election as to avoid swapping out everybody on the board. Therefore, we will be selecting 3 new board members this time around, for a period of 1 or 2 years. An additional change we made last year was to introduce the [nomination committee](https://dotnetfoundation.org/about/election) to help with the process, and to narrow down the final list of candidates. 
```

Comment:
Suggested:

"We've made a few changes over the years as we've learned and grown with the community's help, and last year we started with a staggered election to avoid swapping out everybody on the board. Therefore, this year we'll select three new board members this time around, for a period of 1 or 2 years. We also decided to introduce a Nomination Committee to help with the process and narrow down the final list of candidates."